lib/routes/gov/hangzhou/crawler.ts

```diff
@@ -0,0 +1,54 @@
+import logger from '@/utils/logger';
+export async function zjzwfwCrawler(item: any, browser: any): Promise<string> {
+    try {
+        const page = await browser.newPage();
+        let response = '';
+        try {
+            let navigationSuccess = false;
+            const navigationAttempt = async (attempt) => {
+                if (attempt >= 3) {
+                    return false;
+                }
+                try {
+                    await page.goto(item.link, {
+                        waitUntil: 'networkidle2',
+                        timeout: 60000,
```

Comment:
https://github.com/DIYgod/RSSHub/blob/a8bf6e6f37603204b628e10f38f53a1a072d72eb/lib/utils/puppeteer.ts#L57-L59
The browser will be terminated by RSSHub before that.